db.js

```diff
@@ -481,6 +480,7 @@ exports.init = function (sbot, config) {
     publish,
     addOOO,
     addOOOBatch,
+    setPost: post.set,
```

Comment:
Which module is expected to call this API? I know you said the below already, but it didn't clarify where is this function called.

> *Furthermore the ooo methods must never call that function, unless they are at the head of the feed (this is why I added setPost, secretstack seems to hook that object / function into oblivion).*

Reply:
I'm using it [here](https://github.com/arj03/ssb-browser-core/blob/2a339814c0c6742a78f3a0e6cb1322a03d93c6fc/feed-replication.js#L46). The idea is to get about, get contacts and lastly latest messages. For latest messages we need to set post so that EBT are able to "continue".

Reply:
Okay, I'm fine if we add this for now. I would appreciate some disclaimers, though, like a TODO linking to an issue in GitHub. `setPost` is a bit breaking encapsulation, because `post` used to be read-only for other modules, now it's read-write. I understand that we need to partially replicate latest messages and then "set" the state so that EBT can notice it, but we need to rethink this logic at some point (soon hopefully).

Reply:
Totally agree, I'll add a warning and not add it to the API so that we know it will be changed. Then we can fix this when we do the replicate using indexes & metafeeds.